src/libs/actions/Report.js

```diff
@@ -828,7 +828,7 @@ const handleUserDeletedLinks = (newCommentText, originalHtml) => {
         return newCommentText;
     }
     const htmlWithAutoLinks = parser.replace(newCommentText);
-    const markdownWithAutoLinks = parser.htmlToMarkdown(htmlWithAutoLinks);
+    const markdownWithAutoLinks = parser.htmlToMarkdown(htmlWithAutoLinks).trim();
     const markdownOriginalComment = parser.htmlToMarkdown(originalHtml);
```

Comment:
```suggestion
    const markdownOriginalComment = parser.htmlToMarkdown(originalHtml).trim();
```
Let's trim `markdownOriginalComment` for consistency.

Reply:
Hi @s77rt , thanks for the quick review! I just tested what you suggested here. I think we should trim `markdownWithAutoLinks` here to get it fixed. About consistency, I trimmed text at line 861. Am I missing something?

Reply:
I'm aware that the critical part to trim is `markdownWithAutoLinks`, let's also trim `markdownOriginalComment` for consistency (we always trim `htmlToMarkdown` output)